az_quickstart.install

```diff
@@ -163,3 +163,44 @@ function az_quickstart_update_92101() {
       ->save(TRUE);
   }
 }
+
+/**
+ * Delete unmodified legacy footer menu blocks to prevent global footer issues.
+ */
+function az_quickstart_update_92102() {
+  if (\Drupal::service('module_handler')->moduleExists('az_global_footer')) {
+    return t('Quickstart Global Footer module already installed.  No footer menu blocks need to be deleted.');
+  }
+  else {
+    $delete_blocks = TRUE;
+
+    // Default config hashes generated from clean install of Quickstart 2.0.18.
+    $footer_menu_blocks = [
+      'block.block.az_barrio_footer_menu_info' => 'UPd_XW4rSu7i2VUmkzxPOWBlNM-ckDDv3p_3YFzTOxw',
+      'block.block.az_barrio_footer_menu_main' => 'fYhWRgj2xwKqlRJMJgsVftrGWu3Py7p6SQoHslS6eks',
+      'block.block.az_barrio_footer_menu_resources' => 'NwIzEthgG1SVgWc7GJKsW_44TaWW4HbyY4sZqLqxEKE',
+      'block.block.az_barrio_footer_menu_social_media' => 'Vk6yulKY8Q3ErOnox11fzDTu4-uBxB1oKTQmcxG94RQ',
+      'block.block.az_barrio_footer_menu_topics' => '5GU2oFYWqdhXrFxFcxxArQXQd3YBskaIdVS1uJZ7kDg',
+    ];
+
+    // Check whether any of the footer menu blocks have been modified.
+    foreach ($footer_menu_blocks as $block => $default_config_hash) {
+      $current_config_hash = \Drupal::service('config.factory')
+        ->get($block)
+        ->get('_core.default_config_hash');
+      if ($current_config_hash !== $default_config_hash) {
+        $delete_blocks = FALSE;
+      }
+    }
+
+    if ($delete_blocks) {
```

Comment:
It could be confused, but don't you need to unset the ones that are different from the $footer_menu_blocks array before you delete them all?

Reply:
I suppose the install will break on any existing active config. NVM